src/hmac.rs

```diff
@@ -79,3 +82,29 @@ where
     let keypair = K::from(secret_key);
     Ok(keypair)
 }
+
+////////////////////////////////////////////////////////////////////////
+/// HMAC-SHA256 based functions
+
+pub fn hmac(key: &[u8], message: &[u8]) -> Digest {
+    let mut hash = Hmac::<sha2::Sha256>::new_from_slice(key).unwrap();
+    hash.update(message);
+    let output: [u8; DIGEST_LEN] = hash.finalize().into_bytes().as_slice().try_into().unwrap();
+    Digest::new(output)
```

Comment:
I kinda like that hmac returns a Digest object, it could of course be a byte array as in hkdf, but eventually I believe this is more consistent, assuming we do the same for simple hash function (that in an upcoming PR should also be wrapper functions in fastcrypto).